Project.toml

```diff
@@ -19,7 +19,7 @@ UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 CodecZlib = "0.5, 0.6, 0.7"
 JSON = "0.21"
 OutputCollectors = "0.1"
-julia = "1.3"
+julia = "1.4"
```

Comment:
Out of curiosity, why?

Reply:
Some changes to `Base` having to do with color.  🙄